src/switch_ivr_bridge.c

```diff
@@ -772,7 +772,9 @@ static void *audio_bridge_thread(switch_thread_t *thread, void *obj)
 #if DEBUG_RTP
 		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_NOTICE, "Audio bridge thread: #18 %p -> %p\n", (void*)session_a, (void*)session_b);
 #endif
+			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_NOTICE, "Parsing next event...\n");
 			switch_ivr_parse_next_event(session_a);
```

Comment:
Probably add a channel var before calling `switch_ivr_parse_next_event`. That channel var e.g. like log_next_event or something will indicate to log the content of the event just to lessen the noise. Then unset it after switch_ivr_parse_next_event return.